clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -91,6 +91,9 @@
                                                          + "elapses the client will resend the request if necessary or fail the request if "
                                                          + "retries are exhausted.";
 
+    public static final String MAX_BLOCK_MS_CONFIG = "max.block.ms";
+    public static final String MAX_BLOCK_MS_DOC = "The configuration controls the maximum time that the consumer will block for";
```

Comment:
This is `CommonClientConfigs`, so we should avoid consumer-specific documentation. The producer also has a max.block.ms config, so if we could come up with a generic way to document the behavior of both, it would make sense to write it here. Otherwise, I'd suggest documenting separately.